test/fix_test.go

```diff
@@ -44,7 +44,7 @@ func TestFix(t *testing.T) {
 
 			args := []string{
 				"--disable-all", "--print-issued-lines=false", "--print-linter-name=false", "--out-format=line-number",
-				"--allow-parallel-runners", "--fix",
+				"--allow-parallel-runners", "--fix", "--path-prefix=mock",
```

Comment:
I'm not sure about this. Right now you just replaces one test with another.
It would be better to have different tests for the cases without and with path prefix